public/js/draw/draw.js

```diff
@@ -38,47 +38,127 @@ function getPosition(elem) {
     return { x: xPosition, y: yPosition };
 }
 
-
 /**
- * In node-mode creates a new node at the position of the click event on the SVG canvas.
- * In path-mode creates an elbow at the position of the click event on the SVG canvas,
-                if the startNode is defined.
- * @param {object} e The mousedown event.
+ * Creates a new node at position (posX, posY) on the SVG canvas.
```

Comment:
Follow the documentation convention to properly document the function parameters.